src/commonMain/kotlin/com/jakewharton/byteunits/ByteUnit.kt

```diff
@@ -9,7 +9,7 @@ package com.jakewharton.byteunits
  * @see BinaryByteUnit
  * @see BitUnit
  */
-sealed interface ByteUnit {
```

Comment:
I'll make it `sealed` again once everything is in commonMain. Currently, we can't mix between source sets.

Reply:
I can't. Actual implementations use that interface to implement it, which the Kotlin compiler does not allow if it's sealed.